lib/membrane_live_audio_mixer.ex

```diff
@@ -101,7 +101,7 @@ defmodule Membrane.LiveAudioMixer do
 
   @impl true
   def handle_pad_added(_pad, _context, %{end_of_stream?: true}),
-    do: raise("Can't add input pad after scheduling end of stream")
+    do: raise("Can't add input pad after end of stream")
```

Comment:
```suggestion
      do:  raise("Can't add input pad after scheduling eos and receiving end of stream on all already connected input pads")
```

And we need to adjust `schedule_eos` notification in module docs accordingly